### PR TITLE
Tweaks for the countdown widget.

### PR DESCRIPTION
**Pull request by Camwyn**
Generalize handling of the JSON_LD (all the cool widgets do it)
[ECP-542] & [ECP-546]

There isn't a bucket in TEC, so I am pulling against B21.01 for now.
Requires ~https://github.com/the-events-calendar/tribe-common/pull/1526~ (merged)

See https://github.com/the-events-calendar/events-pro/pull/1656 for artifacts.
🎥 http://p.tri.be/xC4s7Z

[ECP-542]: https://theeventscalendar.atlassian.net/browse/ECP-542
[ECP-546]: https://theeventscalendar.atlassian.net/browse/ECP-546

### CODE REVIEW

File: src/Tribe/Views/V2/Widgets/Service_Provider.php [hunk 1]
```diff
@@ -117,17 +116,4 @@ public function add_views( $views ) {
...
-	 *
-	 * @since 5.3.0
-	 *
-	 * @param array<string,mixed> $template_vars The current template variables.
-	 *
-	 * @return array<string,mixed> The modified template variables.
-	 */
-	public function filter_template_vars( $template_vars ) {
-		return tribe( Widget_List::class )->disable_json_data( $template_vars );
-	}
```
Comment (bordoni) [inline src/Tribe/Views/V2/Widgets/Service_Provider.php:132]: I thought we needed to remove JSON-LD for widgets.

**Review by bordoni — CHANGES_REQUESTED** [review]

File: src/Tribe/Views/V2/Widgets/Service_Provider.php (hunk 1 above)
Comment (Camwyn) [reply src/Tribe/Views/V2/Widgets/Service_Provider.php:132]: Hm, I didn't remove this intentionally...looking.

**Review by lucatume — CHANGES_REQUESTED** [review]
Check the tests.

**Review by bordoni — DISMISSED** [review]

**Comment by borkweb** [comment]
This should be a draft PR until the tests are passing.

**Comment by Camwyn** [comment]
views_widgets tests locally:
Screen Shot 2021-01-14 at 10 41 26 AM

**Review by bordoni — APPROVED** [review]

**Pull request merged**